package.json

```diff
@@ -1,31 +1,45 @@
 {
-  "name": "passrapi",
-  "version": "1.0.0",
-  "description": "Passr API",
-  "main": "index.js",
-  "scripts": {
-    "build:app": "tsc -p tsconfig.json",
-    "build:docker": "docker build . -t passrapi",
-    "build": "npm run build:app && npm run build:docker",
-    "start": "docker run --rm -d -p 5000:5000 passrapi",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Vikelabs/PassrAPI.git"
-  },
-  "author": "See Contributors.md",
-  "license": "SEE LICENSE IN LICENSE",
-  "bugs": {
-    "url": "https://github.com/Vikelabs/PassrAPI/issues"
-  },
-  "homepage": "https://github.com/Vikelabs/PassrAPI#readme",
-  "dependencies": {
-    "express": "^4.17.1"
-  },
-  "devDependencies": {
-    "@types/express": "^4.17.9",
-    "@types/node": "^14.14.14",
-    "typescript": "^4.1.3"
-  }
+	"name": "passrapi",
+	"version": "1.0.0",
+	"description": "Passr API",
+	"main": "index.js",
+	"scripts": {
+		"build:docker": "docker build . -t passrapi",
+		"build": "npm run build:docker",
+		"start:docker": "docker-compose up --build",
+		"start:dev": "nodemon --config nodemon.json src/index.ts",
+		"start": "npm run start:dev",
```

Comment:
is this what runs when i call `npm start`?

Reply:
Yes! Line 11 just executes line 10 since I think the nodemon behaviour is what we will want to develop with.